.vsts-ci.yml

```diff
@@ -0,0 +1,56 @@
+phases:
+- template: /eng/build.yml
+  parameters:
+    agentOs: Windows_NT
+    queue:
+      ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+        name: Hosted VS2017
+      ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:        
+        name: DotNetCore-Build
+        demands:
+        - agent.os -equals Windows_NT        
+      parallel: 2
+      matrix:
+        Build_Debug:
+          _BuildConfig: Debug
+          _PublishType: none
+          _SignType: test
+        Build_Release:
+          _BuildConfig: Release
+          ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+            _PublishType: none
+            _SignType: test
+            PB_PublishBlobFeedKey:
+            PB_PublishBlobFeedUrl:
+          ${{ if notIn(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+            _PublishType: blob
+            _SignType: real
+
+
+- template: /eng/build.yml
+  parameters:
+    agentOs: Linux
```

Comment:
Do you intend to add OSX later?

Reply:
I was waiting for you to tell me the queue I should use for it 😄

Reply:
Nice.  Yesterday I was told that the queue would be available today, but I'm not sure how confident that timeline is.  I have a meeting with Ilya in about 45 minutes and I'll ask how that work is progressing.  When it's populated, the queue will be "DotNetCore-Mac", but there's no machines there at the moment.